src/components/touchableOpacity/index.tsx

```diff
@@ -155,12 +159,12 @@ class TouchableOpacity extends PureComponent<Props, {active: boolean}> {
     );
   }
 
-  onPress(event: BaseSyntheticEvent) {
-    this.props.onPress?.({...this.props, ...event});
+  onPress(event: GestureResponderEvent) {
```

Comment:
Why you changed it from `BaseSyntheticEvent` to `GestureResponderEvent`?

Reply:
this is the correct typing in React Native:
```
    onLongPress?: ((event: GestureResponderEvent) => void) | undefined;

    /**
     * Called when the touch is released,
     * but not if cancelled (e.g. by a scroll that steals the responder lock).
     */
    onPress?: ((event: GestureResponderEvent) => void) | undefined;

    onPressIn?: ((event: GestureResponderEvent) => void) | undefined;

    onPressOut?: ((event: GestureResponderEvent) => void) | undefined;
```

Reply:
OK, please just verify it's the same type for web events (you can run the web demo with this change). 
I thought you picked the `BaseSyntheticEvent` last time for that reason